static/js/settings_account.js

```diff
@@ -198,7 +196,7 @@ exports.initialize_custom_user_type_fields = function (element_id, user_id, is_e
         // pill container for that field and proceed further
         if (field.type === field_types.USER.id && (field_value_raw || is_editable)) {
             const pill_container = $(element_id).find('.custom_user_field[data-field-id="' +
-                                         field.id + '"] .pill-container').expectOne();
```

Comment:
as Tim already mentioned above, please fix these indentation changes, probably your editor has made them, so disable that editor setting first, then fix these changes and amend this commit.

Reply:
I think this has been done to keep the line size below the limit set. Checked in master too, and this is the way it is. What say?